plugin.video.ted-talks/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.ted.talks" name="TED Talks" version="4.2.10" provider-name="rwparris2, moreginger">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
+    <import addon="script.module.elementtree" version="1.2.7"/>
+    <import addon="script.module.parsedom" version="0.9.1"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <!-- There are subtitles on TED in many languages,
+    and this plugin is (partially) internationalized in several,
+    but it seems better to claim the one we are sure of for now. -->
+    <language>en</language>
+    <summary lang="en">Watch talks from TED.com, a site dedicated to 'Ideas Worth Spreading'</summary>
```

Comment:
Just so that I'm clear, what's the fallback logic? I assume it will fallback. Like this?

"en-GB": "en-GB" > "en-EN" > "en-US" (or other, arbitrary order)
"en-EN": "en-EN" > "en-GB" (or other, arbitrary order)

Or does it fall back to the untranslated string (msgid)?